src/clarity-angular/tabs/vtabs-content.ts

```diff
@@ -0,0 +1,42 @@
+import {Component} from "@angular/core";
+
+
+/**
+ * @class VTabContent
+ *
+ * @description
+ * A child component of VTabs that:
+ * 1. Provides an id property set and used by the VTabs class to map VTabsNavLink items to the corresponding
+ *    VTabsContent items
+ * 2. Provides an activated property that triggers Angular to add/remove the content to the DOM via *ngIf
+ */
+@Component({
+    selector: "clr-vtabs-content",
+    template: `
+        <section class="vtabs-content" *ngIf="activated" role="tabpanel">
```

Comment:
As regards `aria-hidden` and accessibility, you should discuss this with @youdz and @adityarb88 because this is a pattern we have used literally _everywhere_. So if the plan is to change it here, you may as well change it everywhere else.

Additionally, we don't use `aria-hidden` to remove content from screenreaders. But ngIf clearly does.

Lastly, if we are only offering hide/show on the angular component, don't you think that's really gimping the HTML/CSS implementation.

You are asking users to come up with their own methods of hiding/showing the active tab content. That's a little much, IMO.

I'm not okay with that.